### ISSUE TITLE
More examples

### ISSUE THREAD

**Issue by kalitine** [jetiny/rollup-plugin-re#2]
I understood better how the `patterns` option works only after reading the issue #1 (I used `match` and `replace` options without `test` and didn't understand why it was not working). I still do not understand the `match` option. Would be nice to have more description in the README.

**Comment by AntonioRedondo** [comment]
Definitively the docs need some brush. It's not clear at all what's the `match` option for.

**Comment by urbanhop** [comment]
yes, doc is urgently needed.
the words "match" and "test" in the examples are perfectly unclear and the docu is just absent.

**Comment by haqer1** [comment]
I agree with all 3 of these comments.
Started with `match`, had to change to `test`.

P.S. Puzzled about about what `match` option does (if it does anything). I think it might be better to just remove `match` from README.md; otherwise adding a clarification on what it does seems _very_ necessary.
P.P.S. On the + side, this plugin worked out better than rollup-plugin-external-globals for what i was after (& i didn't have to do my own IO work-around, which i had kind of started to do).
